packages/client/src/views/SearchResult/AdvancedSearch.tsx

```diff
@@ -93,8 +106,17 @@ export const isAdvancedSearchFormValid = (value: IBaseAdvancedSearchState) => {
       )
   )
 
+  const isInvalidDateField = dateFieldTypes
+    .map((key) => {
+      const exactDate = (
+        value[key as keyof IBaseAdvancedSearchState] as IDateRangePickerValue
+      )?.exact
+      return exactDate && isInvalidDate(exactDate)
+    })
+    .filter((valid) => valid)
+
```

Comment:
Should this be built inside `isValidDateRangePickerValue`?

Reply:
```ts
const validationResultsPerDateField  = dateFieldTypes.map(/* validate */)

const validDateFields = validationResultsPerDateField.map((valid) => valid)
const invalidDateFields = validationResultsPerDateField.map((valid) => !valid)
```

Would something like this help? The way I read `isValidDateRangePickerValue` is it takes in a field and returns if its a valid value or not